launch_xml/README.md

```diff
@@ -0,0 +1,95 @@
+# launch_xml
+
+This package provides an abstraction of the XML tree.
+
+## XML front-end mapping rules
+
+### Accessing xml attributes
+
+When having an xml tag like:
+
+```xml
+<tag attr="2"/>
```

Comment:
@ivanpauno nit: can we change `attr` to some other string to avoid `get_attr('attr', ...)` below?